### ISSUE TITLE
ExpandIcon is always black

### ISSUE THREAD

**Issue by envoy93** [flutter/flutter#18992]
I tried to use the `ExpansionPanelList` and found that the `ExpandIcon` always has `Colors.black38` color:
https://github.com/flutter/flutter/blob/2a65505e8af78aba79ccbae4f0b2bb83bf3c65d4/packages/flutter/lib/src/material/expand_icon.dart#L109
```
    return new IconButton(
      padding: widget.padding,
      color: Colors.black38,
      onPressed: widget.onPressed == null ? null : _handlePressed,
      icon: new RotationTransition(
        turns: _iconTurns,
        child: const Icon(Icons.expand_more)
      )
    );
```
![image](https://user-images.githubusercontent.com/3830719/42163215-bc0cc902-7e12-11e8-92f3-d97e7e417475.png)
That can be a problem if you use a dark theme. I think it's better to use the icon color from the current theme or create different constants for the dark and light themes (like the material buttons):
https://github.com/flutter/flutter/blob/cc1cf9e12a226ec67875a857c2fcc097541781b1/packages/flutter/lib/src/material/flat_button.dart#L242

I'm new to the Flutter, so I can be wrong :)
